### PR TITLE
[ISSUE #1774, Nuget/Home] Update nuget.exe help for high level commands

### PR DESCRIPTION
**Pull request by mido3ds**
PR for issue https://github.com/NuGet/Home/issues/1774

### CODE REVIEW

File: src/NuGet.Clients/NuGet.CommandLine/NuGetCommand.resx [hunk 1]
```diff
@@ -137,7 +137,7 @@ nuget.config HTTP_PROXY</value>
     <comment>Please don't localize this string</comment>
   </data>
   <data name="DeleteCommandDescription" xml:space="preserve">
-    <value>Deletes a package from the server.</value>
```
Comment (nkolev92) [inline src/NuGet.Clients/NuGet.CommandLine/NuGetCommand.resx:140]: When you edit a resource file, please make sure the generated Designer file is added to the PR too.

File: src/NuGet.Core/NuGet.CommandLine.XPlat/Strings.resx [hunk 2]
```diff
@@ -130,7 +130,7 @@
     <value>y</value>
   </data>
   <data name="Delete_Description" xml:space="preserve">
-    <value>Deletes a package from the server.</value>
```
Comment (nkolev92) [inline src/NuGet.Core/NuGet.CommandLine.XPlat/Strings.resx:133]: Same here. 
Please make sure that the relevant Designer is updated as well.

**Review by nkolev92 — CHANGES_REQUESTED** [review]
Hi @mido3ds,

Thank you for your contribution!

I have added some comments to the PR.
Please address them before we proceed.

**Comment by mido3ds** [comment]
@nkolev92
I have update the Designer and added it to PR.  

thanks for your time

**Comment by emgarten** [comment]
Thanks for the PR @mido3ds, I am closing due to lack of activity at this time.

**Pull request closed without merging**